static/templates/settings/account_settings.hbs

```diff
@@ -2,150 +2,154 @@
     <div class="alert" id="dev-account-settings-status"></div>
     <div class="account-settings-form">
         <div class="inline-block">
-            <form class="email-change-form grid">
+            <div id="user_details_section">
```

Comment:
So, the template changes here are pretty hard to read.  What do you think about splitting this into a first commit that just adds this later of divs like `user_details_screen` re-indents everything to be inside those divs (which should be nicely reviewable because it's a simple transformation)), and then a second commit that makes the actual code changes, adding the save/discard widgets (etc.)?  That would make the probably <20 lines of actual changes stand out nice and visibly in the second commit.

Reply:
Done.

Reply:
> iiuc the location of a few blocks have been shifted and then similar settings have been contained together in a div?
> Maybe the re arrangement could be the first commit and wrapping inside `div`s could be the second commit. (so it would be easier to check for any copy paste errors).

Yeah. I'll do that. Thanks.